example/lib/app/widget/app_theme_manager.dart

```diff
@@ -51,7 +51,7 @@ class _AppThemeManagerState extends State<AppThemeManager>
       darkAppThemeData: darkAppThemeData,
       mode: _mode,
       child: AnnotatedRegion<SystemUiOverlayStyle>(
-        value: _mode == AppThemeMode.light
+        value: _mode == AppThemeMode.dark
```

Comment:
Is this right?

Reply:
Yep, if theme mode is dark, status bar must be light (and vice versa).

Reply:
Yes, makes sense